test/spec/functional/viewAttach.js

```diff
@@ -0,0 +1,277 @@
+var testSrcPath = '../../source',
+    spyOnBackbone = require('./backboneSpy');
+
+describe('A View being deactivated and activated', function() {
+
+  var env, _, $, View;
+
+  beforeEach(function(done) {
+    require('./clientEnv')().done(function(environment) {
+      env = environment;
+      $ = env.window.$;
+      _ = env.window._;
+      View = env.window.Torso.Views.View;
+      $('body').append('<div class="app"></div>');
+      done();
+    });
+  });
+
+  var setUpView = function() {
```

Comment:
is this the same setup as the previous test?  Can we have a test utility that exposes these setups?